full.async/src/full/async.clj

```diff
@@ -78,6 +104,7 @@
         ; doall to check for throwables right away
         (doall)))
 
+;; TODO lazy-seq vs. vector in <<! ?
```

Comment:
Sure, this is an option. I was just looking for consistency issues between double-bang and single-bang versions.